crates/holochain/src/core/workflow/sys_validation_workflow.rs

```diff
@@ -1,33 +1,32 @@
 //! The workflow and queue consumer for sys validation
 
-use crate::conductor::space::Space;
 use crate::core::queue_consumer::TriggerSender;
 use crate::core::queue_consumer::WorkComplete;
-use crate::core::sys_validate::check_and_hold_store_record;
 use crate::core::sys_validate::*;
 use crate::core::validation::*;
 use crate::core::workflow::error::WorkflowResult;
-use crate::core::workflow::sys_validation_workflow::validation_batch::{
-    validate_ops_batch, NUM_CONCURRENT_OPS,
-};
 use futures::FutureExt;
+use futures::StreamExt;
 use holo_hash::DhtOpHash;
 use holochain_cascade::Cascade;
 use holochain_cascade::CascadeImpl;
+use holochain_cascade::CascadeSource;
 use holochain_p2p::HolochainP2pDnaT;
 use holochain_sqlite::prelude::*;
-use holochain_state::host_fn_workspace::HostFnStores;
-use holochain_state::host_fn_workspace::HostFnWorkspace;
 use holochain_state::prelude::*;
+use parking_lot::Mutex;
 use rusqlite::Transaction;
 use std::convert::TryInto;
 use std::sync::Arc;
 use tracing::*;
 use types::Outcome;
 
+use self::validation_deps::ValidationDependencies;
+use self::validation_deps::ValidationDependencyState;
+
 pub mod types;
 
-mod validation_batch;
```

Comment:
No longer required, batching is done in place because it's so much less code being run at once now